group_project_v2/messages.py

```diff
@@ -0,0 +1,59 @@
+from group_project_v2.utils import gettext as _
+
+# Generic messages
+UNKNOWN_ERROR = _(u"Unknown error")
+COMPONENT_MISCONFIGURED = _(
+    u"This component is misconfigured and can't be displayed. It needs to be fixed by the course authors."
+)
+STAGE_NOT_OPEN_TEMPLATE = _(u"Can't {action} as stage is not yet opened.")
+STAGE_CLOSED_TEMPLATE = _(u"Can't {action} as stage is closed.")
+
+# Group Project XBlock messages
+NO_ACTIVITIES = _(u"This Group Project does not contain any activities")
```

Comment:
@e-kolpakov Nit: I don't know if there are specs that determine the exact contents of each of the messages listed here, but maybe you can add a full stop to this message and any messages below that don't have one to make them consistent? (Based on their contents, it would make sense to leave `STAGE_URL_NAME_TEMPLATE` and `FAILED_UPLOAD_MESSAGE_TPL` unchanged.)